### PR TITLE
* Fixed attachments for card

### PR DESCRIPTION
**Pull request by harish86**
* Fixed realtime updates to the card

### CODE REVIEW

File: web/static/js/app.js [hunk 1]
```diff
@@ -106,6 +106,35 @@ const VueCurrentUser = {
   }
 };
 
+window.toCamel = function(o) {
```
Comment (harish86) [inline web/static/js/app.js]: @parallel588 I wrote this function to camelcase payload data from websocket. Do you think of any idea how to use VueResourceCaseConverter instead?

Comment (antoineherzog) [inline web/static/js/app.js]: @parallel588 any idea?

Comment (parallel588) [inline web/static/js/app.js]: Hi,  i think we can import methods from VueResourceCaseConverter and use insted window.toCamel.
I'm checking it now. I'll give you know in during hour.

File: web/static/components/cards/show.vue [hunk 2]
```diff
@@ -227,29 +258,32 @@
       'add-contact': AddContact
     },
     methods: {
+      activeContact() {
```
Comment (parallel588) [inline web/static/components/cards/show.vue]: Hi @harish86 , I think we can move this to the computed methods.
https://vuejs.org/v2/guide/computed.html

File: web/static/js/app.js (hunk 1 above)
Comment (parallel588) [inline web/static/js/app.js]: I didn't find how we can use VueResourceCaseConverter  for websocket.
but instead window.toCamel  we can try use vue filter or directive

Comment (harish86) [inline web/static/js/app.js]: @parallel588 I added Vue mixin, and it works.

File: web/static/components/cards/show.vue (hunk 2 above)
Comment (harish86) [inline web/static/components/cards/show.vue]: done, please review

**Comment by antoineherzog** [comment]
@harish86 @parallel588 LGTM Maxim ?

File: web/static/components/cards/show.vue [hunk 3]
```diff
@@ -337,32 +377,39 @@
...
-            this.loadContactInfo();
-            break;
-          }
-        }
+        let url = '/api/v2/contact/' + this.contacts[this.activeContactIndex].id;
+        this.$http.put(url, { contact: this.contacts[this.activeContactIndex] } );
       },
 
       removeContact(contactId) {
+        console.log(contactId);
```
Comment (parallel588) [inline web/static/components/cards/show.vue]: i think we can remove it

Comment (harish86) [inline web/static/components/cards/show.vue]: @parallel588 updated

**Pull request merged**